src/main/java/com/interview/library/service/BookService.java

```diff
@@ -50,6 +51,17 @@ public Book save(Book book) {
         return bookRepository.findAll(pageable);
     }
 
+    /**
+     * Get all books its price is lower than <i>priceThreshold</i> parameter.
+     *
+     * @param pageable the pagination information
+     * @return the list of entities
+     */
+    @Transactional(readOnly = true)
+    public Page<Book> findAllEqualOrLessThan(Pageable pageable, BigDecimal priceThreshold) {
+        log.debug("Request to get all Books its price is under " + priceThreshold);
```

Comment:
`Books its price` -> `Books where the price`

Reply:
Changed (modified it a bit since it contained more errors)

Reply:
ok